p/scripts/category.js

```diff
@@ -47,41 +47,53 @@ function init_draggable() {
 	const dropSection = document.querySelector('.drop-section');
 
 	dropSection.ondragstart = function (ev) {
-		const li = ev.target.closest ? ev.target.closest(draggable) : null;
-		if (li) {
+		const li_draggable = ev.target.closest ? ev.target.closest(draggable) : null;
+		if (li_draggable) {
+			const ulClosest = li_draggable.closest('ul');
+			ulClosest.classList.add('disallowDragging');
```

Comment:
I'd say something more like `drag-disallowed` or simply `no-drag`, for consistency with `drag-active` and `drag-hover`.

Reply:
thanks. Fixed it